### PR TITLE
Updated vaccine external data code

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Hey @agitter, this builds on top of #1149, so it will look very messy here, but it's the proof-of-concept for automatically generating text that updates in response to external data. I figured I would open it as a draft since we were talking about it on email, but I don't except you to review it now since it's part of a bigger modification to the pipeline.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — COMMENTED** [review]
Very cool idea.  Is `owiddata/05.autogenSentences.py` the only new part of this pull request with everything else being the changes introduced in #1149?

**Comment by rando2** [comment]
I will remove the .gitignore and try to reconcile the files before merging. Here are the new graphs:





(Didn't end up using the second one because it seems a little weird, but I am hoping to find another data source to separate out by high/low/middle income countries)

**Comment by agitter** [comment]
@rando2 I can review the code changes.  Because this builds on a prior pull request, can you please direct me to which files changed that I should focus on?

These data and new graphics are very interesting.

**Comment by rando2** [comment]
> @rando2 I can review the code changes. Because this builds on a prior pull request, can you please direct me to which files changed that I should focus on?
> 
> These data and new graphics are very interesting.

Hi Tony, Thank you so much! The only major changes are in 05 and 06-- the rest are just some small edits for refactoring. Good luck with the grant, and I don't think this PR is urgent!

File: owiddata/04.drawMaps.py [hunk 1]
```diff
@@ -2,26 +2,11 @@
 from jsonFunctions import *
 import pandas as pd
 import numpy as np
-import geopandas
+from mapFunctions import *
```
Comment (cgreene) [inline owiddata/04.drawMaps.py:5]: Can you import by name, not star? If nothing else, for review ease :)

File: owiddata/05.autogenSentences.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import argparse
+import pandas as pd
+from jsonFunctions import *
```
Comment (cgreene) [inline owiddata/05.autogenSentences.py:3]: As above

File: owiddata/06.plotVaxCountryData.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import argparse
+import pandas as pd
+from jsonFunctions import *
+from mapFunctions import *
+from plotnine import *
```
Comment (cgreene) [inline owiddata/06.plotVaxCountryData.py:5]: star import may be ok here if it follows the package norms

**Review by cgreene — APPROVED** [review]
A few comments here and there - not sure if Tony wants to look as well.

**Review by agitter — APPROVED** [review]
I looked through at a high level and am okay merging.  If there are any issues, we can fix them post-merge or I'll look for them as I'm reviewing the vaccines manuscripts.

**Comment by rando2** [comment]
Thank you so much @agitter!

**Pull request merged**